pom.xml

```diff
@@ -42,6 +42,8 @@
   <url>http://blueflood.io</url>
 
   <properties>
+    <VERSION_SUFFIX>0-SNAPSHOT</VERSION_SUFFIX>
+    <BLUEFLOOD_VERSION>2.0.${VERSION_SUFFIX}</BLUEFLOOD_VERSION>
     <maven.javadoc.skip>true</maven.javadoc.skip>
```

Comment:
We can avoid having two variables by doing something similar to the command below. We should not checkin the files changed. After we change versions we can run the "mvn package" command and tag the release.

```
mvn build-helper:parse-version versions:set -DnewVersion=${parsedVersion.majorVersion}.\${parsedVersion.minorVersion}.\${parsedVersion.incrementalVersion}-${BUILD_NUMBER}
```

I found this link helpful which is similar to what you are doing. https://www.thoughtworks.com/insights/blog/continuous-delivery-maven-and-go-maven-central